stdlib/tests/word/mod.rs

```diff
@@ -0,0 +1,34 @@
+use core::iter;
+
+use miden_core::{Felt, Word};
+use miden_utils_testing::{ONE, StackInputs, ZERO, felt_slice_to_ints, rand::seeded_word};
+
+#[test]
```

Comment:
I think we could move the `gt`/`lt` tests from miden-base here: https://github.com/0xMiden/miden-base/blob/8e88dc6b04429f67c685ce577f2afce6f8df757a/crates/miden-testing/src/kernel_tests/tx/test_link_map.rs#L370.

Reply:
Excellent idea.